rust/stackable-cockpit/src/platform/stack/spec.rs

```diff
@@ -1,75 +1,98 @@
 use std::collections::HashMap;
 
 use serde::{Deserialize, Serialize};
-use snafu::{ResultExt, Snafu};
+use snafu::{OptionExt, ResultExt, Snafu};
 use tracing::{debug, info, instrument, log::warn};
 
 #[cfg(feature = "openapi")]
 use utoipa::ToSchema;
 
 use crate::{
-    common::ManifestSpec,
-    helm::{self, HelmChart, HelmError},
+    common::manifest::ManifestSpec,
+    helm,
     platform::{
         cluster::{ResourceRequests, ResourceRequestsError},
         demo::DemoParameter,
-        release::{ReleaseInstallError, ReleaseList},
+        release,
     },
     utils::{
-        k8s::{KubeClient, KubeClientError},
+        k8s,
         params::{
             IntoParameters, IntoParametersError, Parameter, RawParameter, RawParameterParseError,
         },
         path::{IntoPathOrUrl, PathOrUrlParseError},
     },
     xfer::{
+        self,
         processor::{Processor, Template, Yaml},
-        FileTransferClient, FileTransferError,
     },
 };
 
 pub type RawStackParameterParseError = RawParameterParseError;
 pub type RawStackParameter = RawParameter;
 pub type StackParameter = Parameter;
 
-/// This error indicates that the stack, the stack manifests or the demo
-/// manifests failed to install.
 #[derive(Debug, Snafu)]
-pub enum StackError {
+pub enum Error {
     /// This error indicates that parsing a string into stack / demo parameters
     /// failed.
-    #[snafu(display("parameter parse error: {source}"))]
+    #[snafu(display("failed to parse demo / stack parameters"))]
     ParameterError { source: IntoParametersError },
```

Comment:
I thought clippy complaints about variants ending on Error. Seems not to be the case anymore, but you tried to remove it in other modules as well?

Reply:
It did? I actually don't know... You are right tho. I will update that error variant to be in line with other variants.

> but you tried to remove it in other modules as well?

Yes that was part of this PR.

Reply:
Thanks. I think it was if the enum was called *Error und a variant was *Error or something, like redundancy.

Reply:
Updated many more error variants: a050a37